spread.yaml

```diff
@@ -9,7 +9,7 @@ environment:
     SNAP_REEXEC: 0
     SPREAD_STORE_USER: "$(HOST: echo $SPREAD_STORE_USER)"
     SPREAD_STORE_PASSWORD: "$(HOST: echo $SPREAD_STORE_PASSWORD)"
-    LANG: "$(echo $LANG)"
+    LANG: "$(echo ${LANG:-C.UTF-8})"
```

Comment:
doesn't this being in the `environment` block set up the variable for child processes?